.env.sample

```
@@ -1,5 +1,6 @@
 MODE=development # development, test, production
 ORIGIN=https://example.com  # base url of server
+PRODUCTION_ACTOR_HANDLE=@yourFedifyId@hackers.pub #if current environments is "development", replace ngrok_fediverse_id into real_fediverse_id that you used. (ngrok_fediverse_id is not found in production)
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)